# Commands
uint8 CMD_FAILSAFE=0
uint8 CMD_LAND=1
uint8 CMD_SETPOINT_POS=2
uint8 CMD_ACTUATE=3

geometry_msgs/PoseStamped[] path
uint8 ending_cmd
